tools/generate_hotkeys.py

```diff
@@ -1,39 +1,148 @@
+                    )
+                    error_flag = 1
+            # Check key combination duplication
+            check_duplicate_keys_list = [
+                key for key in check_duplicate_keys_list if key not in KEYS_TO_EXCLUDE
+            ]
+            duplicate_keys = [
+                key
+                for key in check_duplicate_keys_list
+                if check_duplicate_keys_list.count(key) > 1
+            ]
+            if len(duplicate_keys) != 0:
+                print(
+                    f"Duplicate key combination for keys {duplicate_keys} for category ({HELP_CATEGORIES[action]}) detected"
+                )
+                error_flag = 1
+        if error_flag == 1:
+            print(
+                "After resolving above errors, Run './tools/generate_hotkeys' to update hotkeys."
+            )
+            print("Again run './tools/generate_hotkeys --check-only' to lint hotkeys.")
+        exit(error_flag)
+    else:
+        print("Run './tools/generate_hotkeys' to update the hot keys")
+
+
+def generate_hotkeys_file() -> None:
+    """
+    Generate hotkeys.md in docs folder based on help text description and shortcut key combinations in config/keys.py file
+    """
+    hotkeys_file_string = get_hotkeys_file_string()
+    compare_hotkeys_file(hotkeys_file_string)
+    with open(OUTPUT_FILE, "w") as mdFile:
+        mdFile.write(hotkeys_file_string)
+        print(f"Hot Keys list saved in {OUTPUT_FILE}")
+
+
+def get_hotkeys_file_string() -> str:
+    """
+    Get existing hotkeys.md in docs folder as string based on help text description and shortcut key combinations in config/keys.py file
+    """
+    categories = read_help_categories()
+    hotkeys_file_string = f"<!--- Generated automatically by tools/{SCRIPT_NAME} -->\n<!--- Do not modify -->\n\n# Hot Keys\n"
+    for action in HELP_CATEGORIES.keys():
+        hotkeys_file_string += (
             f"## {HELP_CATEGORIES[action]}\n"
             "|Command|Key Combination|\n"
             "| :--- | :---: |\n"
         )
-        for help_text, key_combinations_list in categories[action].items():
-            various_key_combinations = " / ".join(
-                [
-                    " + ".join([f"<kbd>{key}</kbd>" for key in key_combination.split()])
-                    for key_combination in key_combinations_list
-                ]
-            )
-            mdFile.write(f"|{help_text}|{various_key_combinations}|\n")
-        mdFile.write("\n")
+        for help_text, key_combinations_list in categories[action]:
+            various_key_combinations = various_key_combination(key_combinations_list)
+            hotkeys_file_string += f"|{help_text}|{various_key_combinations}|\n"
+        hotkeys_file_string += "\n"
+    return hotkeys_file_string
+
+
+def compare_hotkeys_file(hotkeys_file_string: str) -> bool:
+    if hotkeys_file_string == open(OUTPUT_FILE).read():
+        print("Hot Keys already in sync with config/keys.py")
```

Comment:
Using `hotkeys.md` instead of `Hot Keys` makes sense in both this and the below print statements as well.